src/components/Compare/CompareTable.tsx

```diff
@@ -211,12 +212,26 @@ const CompareTable = (props: {
     trackCompareEvent(EventAction.OPEN_MODAL, 'Show All Locations');
   };
 
+  function getLocationLimit() {
```

Comment:
this function does the same thing as [the second half of this function](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Compare/LocationTable.tsx#L253). how about we combine them?